### DISCUSSION TITLE
Soundboard ?

### DISCUSSION THREAD

**Discussion by jCOTINEAU** [quodlibet/quodlibet#4594] [Ideas]
Hello everyone, I discovered Quodlibet and started using it for handling music during TTRPG's sessions.

The tool is really cool and I would like to know if it could make sense to build a "soundboard" into it.

By that, I mean a panel with the opportunity to mix various sounds and play them together, trigger them with shortcuts, have them on loops, etc.

I Would be happy to contribute to it, but I don't know where to start. Could you guys help me and point to a high-level overview of the components in the code I should look at?

Could this be done with a plugin? 

Let me know what you think. Thank you very much in advance for the tool!

**Comment by declension** [comment]
As a plugin, this would be awkward but doable IMO. It doesn't sound that related to QL in general so I'm not sure the benefit of having it in there, unless it's the benefit of having the library metadata and searching etc for all the samples (which makes sense)

**Comment by jCOTINEAU** [comment]
Yes so far the searching capabilities of QL are so powerful it is tempting for a soundboard, nonetheless I understand it might not be that related, thanks a lot for the discussion.
